src/llv8.cc

```diff
@@ -457,6 +458,11 @@ std::string HeapObject::Inspect(bool detailed, Error& err) {
     return pre + fn.Inspect(detailed, err);
   }
 
+  if (type == v8()->types()->kJSRegExpType) {
+    JSRegExp re(this);
+    return pre + re.Inspect(detailed, err);
```

Comment:
Yep, there are others that do not work like this. Want me to include those in this same PR?